### PR TITLE
IBX-7611: Bump version for webpack and sass-loader

### PR DESCRIPTION
**Pull request by lucasOsti**
Jira ref: https://issues.ibexa.co/browse/IBX-7611

Bump version for 
 - @symfony/webpack-encore ^1.7.0 -> ^3.0.0
 - sass-loader ^12.5.0 -> ^13.0.0

### CODE REVIEW

**Review by glye — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

File: ibexa/content/4.5/encore/package.json [hunk 1]
```diff
@@ -2,14 +2,14 @@
   "devDependencies": {
     "@hotwired/stimulus": "^3.0.0",
     "@symfony/stimulus-bridge": "^3.0.0",
-    "@symfony/webpack-encore": "^1.7.0",
+    "@symfony/webpack-encore": "^3.0.0",
```
Comment (dew326) [inline ibexa/content/4.5/encore/package.json:5]: Why we are changing it for 4.5 in content but not for other editions?

File: ibexa/experience/4.6/encore/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "devDependencies": {
     "@hotwired/stimulus": "^3.0.0",
     "@symfony/stimulus-bridge": "^3.0.0",
-    "@symfony/webpack-encore": "^1.7.0",
+    "@symfony/webpack-encore": "^3.0.0",
```
Comment (dew326) [inline ibexa/experience/4.6/encore/package.json:5]: missing sass-loader change

**Review by dew326 — CHANGES_REQUESTED** [review]

**Review by dew326 — APPROVED** [review]

**Comment by glye** [comment]
@lucasOsti Ready for QA?

**Review by glye — CHANGES_REQUESTED** [review]
Oh BTW I just noticed the target is master, this should go into 4.5

**Comment by lucasOsti** [comment]
Here we only have branch master but in this PR I edited files for version 4.5 e.g https://github.com/ibexa/recipes-dev/pull/108/files#diff-b602eecefcd395939d9c80eec7bc0f8a40fbc0d53a52537e12d7e81415dff1baL2

**Review by glye — APPROVED** [review]
D'oh 😅

**Review by micszo — APPROVED** [review]
Installed and tested on 4 editions of 4.5 and 4.6.

**Pull request merged**